docs/G-Codes.md

```diff
@@ -704,10 +704,13 @@ The following commands are available when an
   the first time, it starts the measurements, next execution stops
   them. If RATE is not specified, then the default value is used
   (either from `printer.cfg` or `3200` default value). The results of
-  measurements are written to a file named `/tmp/adxl345-<name>.csv`
-  where `<name>` is the optional NAME parameter. If NAME is not
-  specified it defaults to the current time in "YYYYMMDD_HHMMSS"
-  format.
+  measurements are written to a file named
+  `/tmp/adxl345-<chip>-<name>.csv` where `<chip>` is the name of the
```

Comment:
If we change the behavior if this command I think we should note the change in the docs/Config_Changes.md file.

Reply:
If it is worth mentioning - sure. I also added the `TEST_RESONANCES` command change there then.